integrations/jira/.port/resources/blueprints.json

```diff
@@ -1,4 +1,83 @@
 [
+  {
+    "identifier": "jiraBoard",
+    "title": "Jira Board",
+    "description": "This blueprint represents a Jira board",
+    "icon": "Jira",
+    "schema": {
+      "properties": {
+        "url": {
+          "title": "Board URL",
+          "type": "string",
+          "format": "url",
+          "description": "URL to the board in Jira"
+        },
+        "type": {
+          "title": "Type",
+          "type": "string",
+          "description": "The type of the board"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "project": {
```

Comment:
Is it possible to have more than 1 project in a Jira board? if so, we need to move this relation to the jiraProject instead. Upon checking it looks like a boar can have multiple projects so we need to remove this relation

https://developer.atlassian.com/cloud/jira/software/rest/api-group-board/#api-rest-agile-1-0-board-boardid-project-get